src/utils/plotSpecs/generateFrequencySpec.js

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 
 import { intersection } from '../cellSetOperations';
 
-const generateSpec = (config, plotData) => {
+const generateSpec = (config, { xNamesToDisplay, yNamesToDisplay, plotData }) => {
```

Comment:
Rather than passing `plotData` as an object, it would be clearer to pass `xNamesToDisplay, yNamesToDisplay` as it's own parameter.